### DISCUSSION TITLE
Is it any way to unwrap  bscDUCO to DUCO ?

### DISCUSSION THREAD

**Discussion by panayotovip** [revoxhere/duino-coin#1378] [Q&A]
Is it any way to unwrap  bscDUCO from MetaMask to DUCO ?
In WiKi have only how to wrap DUCO to bscDUCO.

**Comment by LDarki** [comment]
Hi, yes.
You can use the [official web site](https://bsc.duinocoin.com/) or [my website](https://ldarki.github.io/bscDUCO).
